### ISSUE TITLE
Timecodes for "ML Zoomcamp Office Hours - Week #11"

### ISSUE THREAD

**Issue by alexeygrigorev** [DataTalksClub/machine-learning-zoomcamp#147]
Youtube video: https://www.youtube.com/watch?v=1WRgdBTUaAc

**Comment by dimzachar** [comment]
0:00:00 - Explanation of training neural network from scratch.
0:02:26 - Sliding filter creates feature map from image.
0:05:18 - Sliding filter to compute similarity, stride control.
0:07:43 - Explaining filter computation and max pooling layer
0:10:02 - Explanation of pooling and flattening layers.
0:12:28 - Using Keras for non-image tabular data.
0:15:48 - Using pipelines to transform data for modeling.
0:19:12 - Scaling data for neural networks using standard scaler.
0:21:46 - Testing input/output of Keras model.
0:24:15 - Building linear regression model with Keras.
0:26:42 - Training a linear regression model.
0:29:15 - Compiling and training Keras model.
0:31:33 - Using tqdm callback to track progress.
0:33:59 - Model training progress and performance analysis.
0:36:25 - Adding inner layer and dropout for improvement.
0:39:35 - Training model with dropout for regularization.
0:41:57 - Defining a sequential model in Keras.
0:44:30 - Comparing functional and sequential model definitions.
0:47:11 - Building neural network to compare images.
0:49:56 - Activation, logits, and pooling explained briefly.
0:52:19 - Explaining 2D and 3D pooling layers
0:55:13 - Explaining functional style coding in ML
0:57:45 - Comparing functional and sequential model building.

**Comment by amitfrancis** [comment]
Updated timecodes. Thanks!
